pandas/util/validators.py

```diff
@@ -0,0 +1,48 @@
+"""
+Module that contains many useful utilities
+for validating data or function arguments
+"""
+
+
+def validate_args(args, min_length=0, max_length=None):
+    """
+    Checks whether the length of the `*args` argument passed into a function
+    has at least `min_length` arguments. If `max_length` is an integer, checks
```

Comment:
Not sure that there actually is a `Returns` section in the documentation as far as I understand but can definitely add a `Parameters` section.